docs/paper/dev/misc/nms.mdx

```diff
@@ -0,0 +1,71 @@
+---
+slug: /dev/nms
+---
+
+# NMS
+
+NMS is an abbreviation for `net.minecraft.server`. It is the package that contains Mojang's classes that are used by the
+server. This is internal code which often changes between versions. For this reason, plugins that use NMS code are
+often version specific or require different code for different versions. It is often used to refer to CraftBukkit code
+as well.
+
+## Using NMS and CraftBukkit
+
+In order to use NMS and CraftBukkit code, you may either add the dependencies to your plugin or use reflection. Depending
```

Comment:
"add the dependencies" is pretty generic, especially considering you mention paperweight below (without saying its adding the dependencies)